src/h5web/dataset-visualizer/mapper/SlicingSlider.tsx

```diff
@@ -0,0 +1,44 @@
+import React, { ReactElement } from 'react';
+import ReactSlider from 'react-slider';
+import styles from './DimensionMapper.module.css';
+import type { DimensionMapping } from '../models';
+
+interface Props {
+  dimension: number;
+  slicingIndex: number;
+  rawDims: number[];
+  mapperState: NonNullable<DimensionMapping>;
+  onChange: (mapperState: DimensionMapping) => void;
+}
+
+function SlicingSlider(props: Props): ReactElement {
+  const { dimension, slicingIndex, rawDims, mapperState, onChange } = props;
+
+  return (
+    <div key={dimension} className={styles.sliderWrapper}>
+      <span className={styles.sliderLabel}>D{dimension}</span>
+      <ReactSlider
+        key={`${mapperState.includes('y')}`}
```

Comment:
Now that this is in a separate component, a comment would be needed to explain the *raison d'être* of this key